ci/cases/C96_atm3DVar_also.yaml

```diff
@@ -0,0 +1,14 @@
+experiment:
```

Comment:
Instead of this, you could retain the one that is deleted.
`C96C48_hybatmDA.yaml`

Reply:
ok